GameServerLib/Logic/GameObjects/AttackableUnits/AI/Champion.cs

```diff
@@ -63,6 +63,12 @@ public class Champion : ObjAiBase
 
             Spells[4] = new Spell(this, clientInfo.SummonerSkills[0], 4);
             Spells[5] = new Spell(this, clientInfo.SummonerSkills[1], 5);
+
+            for(short i = 0; i < 7 ; i++)
```

Comment:
How about doing
```cs
for(byte i = 6; i < 13; i++)
{
    Spells[i] = new Spell(this, "BaseSpell", i);
}
```
instead?

Reply:
I wrote it the way I did for consistency with the other spell loads, but now that you mention it there probably isn't much of a reason to not do it the way you suggested. Will update in a moment.